oapth/src/fixed_sql_commands/pg.rs

```diff
@@ -26,30 +26,50 @@ where
   for schema in schemas(back_end).await? {
     buffer.write_fmt(format_args!("DROP SCHEMA {} CASCADE;", schema))?;
   }
-  
+
   for domain in domains(back_end, "public").await? {
     buffer.write_fmt(format_args!("DROP DOMAIN {};", domain))?;
   }
-  
+
   for function in functions(back_end, "public").await? {
     buffer.write_fmt(format_args!("DROP FUNCTION {} CASCADE;", function))?;
   }
-  
+
   for table in back_end.tables("public").await? {
     buffer.write_fmt(format_args!("DROP TABLE {} CASCADE;", table))?;
   }
-  
+
   for procedure in procedures(back_end, "public").await? {
     buffer.write_fmt(format_args!("DROP PROCEDURE {} CASCADE;", procedure))?;
   }
-  
+
   for ty in types(back_end, "public").await? {
     buffer.write_fmt(format_args!("DROP TYPE {} CASCADE;", ty))?;
   }
 
+  for sequence in sequences(back_end, "public").await? {
+      buffer.write_fmt(format_args!("DROP SEQUENCE {};", sequence))?;
+  }
+
   Ok(buffer)
 }
 
+// https://github.com/flyway/flyway/blob/master/flyway-core/src/main/java/org/flywaydb/core/internal/database/postgresql/PostgreSQLSchema.java
+#[oapth_macros::dev_tools_]
+#[inline]
+pub async fn sequences<B>(back_end: &mut B, schema: & str) -> crate::Result<Vec<String>>
+where
+ B: crate::BackEnd
+{
+    let mut buffer = ArrayString::<[u8; 128]>::new();
+    buffer.write_fmt(format_args!(
+            "SELECT sequence_name FROM information_schema.sequences WHERE sequence_schema = '{schema}'",
```

Comment:
```suggestion
            "SELECT sequence_name  AS generic_column FROM information_schema.sequences WHERE sequence_schema = '{schema}'",
```

Column aliasing is necessary in this case because the diesel back-end expects a compile-time table definition to actually run queries.

https://github.com/c410-f3r/oapth/blob/master/oapth/src/back_end_impls/diesel/schemas.rs#L7